server/ansible/roles/pbench-clean-yum-cache/tasks/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: Clean yum cache
+  command: yum clean all
+
```

Comment:
Can we also add `rm -rf /var/cache/yum`?

Probably with the "directory" / "absent" file module?

Reply:
Sure.